packages/components/src/date-time/test/date.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import DatePicker from '../date';
+
+describe( 'DatePicker', () => {
```

Comment:
I'm glad to see tests, but for what it's worth these tests as written would not have caught the error described in my previous review, since there is zero coverage for `onChangeMoment`.